pkg/cri/server/events.go

```diff
@@ -187,7 +188,12 @@ func (em *eventMonitor) startContainerExitMonitor(ctx context.Context, id string
 
 				cntr, err := em.c.containerStore.Get(e.ID)
 				if err == nil {
-					if err := handleContainerExit(dctx, e, cntr, em.c); err != nil {
+					sandboxID := cntr.SandboxID
+					sb, err := em.c.sandboxStore.Get(sandboxID)
+					if err != nil {
+						return fmt.Errorf("can't find sandbox for sandbox ID %s: %w", sandboxID, err)
```

Comment:
same here.. let's just log and continue.. verify that kubelet can receive the event notifying them that the activity occurred but no podsandbox...

Reply:
on the log output let's call these podsandbox vs sandbox.. unless it is a sandbox of a different type.. iow .. cri podsandbox is not equivalent to sandbox of a different service type in containerd and over time the  new sandbox types mixing with pod sandboxes will be confusing